README.md

```diff
@@ -51,7 +51,7 @@ A WordPress plugin providing a thin wrapper around Wikipedia Preview to simplify
   wikipediaPreview.init()
 </script>
 ```
-You can serve the file yourself or include it from [unpkg](https://unpkg.com/browse/wikipedia-preview@latest/dist/wikipedia-preview.umd.cjs).
```

Comment:
Is it worth exploring the workaround mentioned here: https://github.com/mjackson/unpkg/issues/268#issuecomment-1541525068

Reply:
right, it's worth adding a quick workaround note as it is unlikely that the Unpkg will merge the PR to address this issue.